contributions/open-source/kallem-mwesslen/README.md

```diff
@@ -0,0 +1,24 @@
+# Open-source contribution proposal: Contributor Scoreboard
+## Members
+Kalle Meurman (kallem@kth.se) Github: Wizkas0
+
+Markus Wesslen (mwesslen@kth.se) Github: m4reko
+
+## Proposal
+We would like to create an open-source project from scratch. Our idea is to create a github action/ workflow, 
+that scores contributors to a repository based on user-defined rulesets. A rule might for instance be that 
+commits should link to an issue. Each contributor would then have a score of (commits with linked issues)/(total commits) 
+and this score would be kept in a scoreboard (perhaps as a pinned issue).
+
+The point of this is to encourage good practices in contributors, which would improve workflow.
+
+The scope of this project is potentially very large, as there are many rules of varying degrees of complexity 
```

Comment:
It's a super interesting idea. However, as you mentioned the scope is too large. Please specify the exact rule that you want to consider.

Reply:
Would the one mentioned suffice? Commits needing to link to issues?

Reply:
That would be enough. However, do you think such a rule actually makes sense? Why should we mention an issue in every single commit? (note that "commit != pull request")

Reply:
True, we'll think about it a bit more. I'll try to find some common rules and see if any of them seems appropriate.

Reply:
Good, I'll be waiting for your update.